portfolio/src/main/webapp/script.js

```diff
@@ -155,3 +155,17 @@ async function checkLogin(){
     document.getElementById("login-link").href = details.loginUrl;
   }
 }
+
+/** Creates a map and adds it to the page. */
+function initMap() {
+  const nightMap = new google.maps.StyledMapType(nightVersion, {name: 'Night'});
```

Comment:
Can you please add a screenshot to the night version of the map? :) I'm excited to see what does it look like :D

Reply:
Hi this is what the map looks like! and yes I make it to be centered in Jakarta, Indonesia :D
standard (day) map
![image](https://user-images.githubusercontent.com/41379082/88702802-bcbf9280-d103-11ea-9b22-d5f913f204ed.png)


night map
![Screenshot 2020-07-28 at 6 54 35 PM](https://user-images.githubusercontent.com/41379082/88702861-dbbe2480-d103-11ea-81a4-2255ccb79ebe.png)

Reply:
Looks great! :D Thanks Rania :)